TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/AcceptedListPageComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedListPageComponent
+    {
+        IWebDriver driver { get; }
```

Comment:
please specify the access modifier explicitly

Reply:
added